pkgs/development/python-modules/rxv/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, defusedxml
+, requests
+, pytest
+, requests-mock
+, mock
+, pytestcov
+, pytest-timeout
+, testtools
+}:
+
+buildPythonPackage rec {
+  pname = "rxv";
+  version = "0.6.0";
+
+  propagatedBuildInputs = [ defusedxml requests ];
+
+  checkInputs = [ pytest requests-mock mock pytestcov pytest-timeout testtools ];
+
+  checkPhase = ''
+    pytest
+  '';
+
+  src = fetchFromGitHub {
+    owner = "wuub";
+    repo = pname;
+    # Releases are not tagged
+    rev = "9b586203665031f93960543a272bb1a8f541ed37";
+    sha256 = "1dw3ayrzknai2279bhkgzcapzw06rhijlny33rymlbp7irp0gvnj";
+  };
```

Comment:
to be consistent with other packages, please re-order:

```
  pname = "rxv";
  version = "0.6.0";

  src = fetchFromGitHub {
    owner = "wuub";
    repo = pname;
    # Releases are not tagged
    rev = "9b586203665031f93960543a272bb1a8f541ed37";
    sha256 = "1dw3ayrzknai2279bhkgzcapzw06rhijlny33rymlbp7irp0gvnj";
  };

  propagatedBuildInputs = [ defusedxml requests ];

  checkInputs = [ pytest requests-mock mock pytestcov pytest-timeout testtools ];
  checkPhase = ''
    pytest
  '';
```

Reply:
Done